### DISCUSSION TITLE
When are scenarios reported as skipped?

### DISCUSSION THREAD

**Discussion by bensullivan** [getgauge/gauge#2151] [General]
Hi

I don't understand when scenarios are reported as skipped in the html report generated by `gauge run`.

Could you clarify the criteria for scenarios to be listed as Skipped in the run report?

Thanks

Ben

**Comment by sriv** [comment]
Hi @bensullivan 

Some cases when scenario can be marked as skipped when:

- scenario has any parse errors
- it is a table driven execution and `--table-rows` flag is specified which filters out a scenario for a particular row
- runner has crashed before the execution of scenario
- scenario data store failed to initialize

Above is from the top of my head, there may a case of two more, need to look at the code to confirm.
